osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -44,6 +44,10 @@ public GlobalActionContainer(OsuGameBase game)
             new KeyBinding(InputKey.Space, GlobalAction.Select),
             new KeyBinding(InputKey.Enter, GlobalAction.Select),
             new KeyBinding(InputKey.KeypadEnter, GlobalAction.Select),
+
+            new KeyBinding(InputKey.F5, GlobalAction.MusicPrev),
```

Comment:
I don't like these keybindings. I would prefer to use Fn combined with special keys for these actions.

Reply:
I first wanted to bind these actions to media keys , but they don't seem to be available in InputKey enum. Guess I'll end up binding them to Fn+x equivalents of media keys play / pause / next and previous

Reply:
This may be a good chance to add those extra entries into `InputKey` (shouldn't be a huge ordeal to do so), though not 100% necessary to merge this.

Reply:
Adding media keys into `InputKey` enum may require also a PR osuTK-side as it doesn't recognise media keys (media keys are commented out in the `Key` enum)

Reply:
Let's use stable keybindings similar to stable's one as using media keys require a PR framework-side & osuTK-side